charts/refinery/templates/deployment.yaml

```diff
@@ -59,6 +60,9 @@ spec:
           {{- toYaml . | nindent 12 }}
           {{- end }}
           ports:
+            - name: debug
+              containerPort: 6060
```

Comment:
Refinery's default behavior is to use the first available port between :6060 and :6069 when `DebugServiceAddr` is not set.  Since the helm chart doesn't set this value by default, we cannot rely on this value always being the port chosen by Refinery.

Reply:
If it's the only thing running in the container, why wouldn't it be able to bind :6060? oh, unless the config is explicitly setting DebugServiceAddr to something else, you're suggesting?

Reply:
Correct. If the user explicitly sets the value hard coding this definitely won't work. If they have chosen to bind something else to port 6060 (via some config value) this assumption also breaks down.